mapdata/WarcraftLegacies/UnitData/Skin/E01D.json

```
@@ -131,7 +131,7 @@
     {
       "Id": 1651864693,
       "Type": 3,
-      "Value": "Mystical Hero, adept at disruption and ranged combat."
+      "Value": "Mystical Hero, adept at disruption and ranged combat.|n|cfff5962dAbilities:|r Void Rift, Hero Glow Origin|n|cfff5962dAbilities (hero):|r Arcane Bombardment, Energy Wave, Mana Burn, Black Hole, Attribute Bonus|n|n|cffffcc00Attacks land and air units.|r"
```

```suggestion
      "Value": "Mystical Hero, adept at disruption and ranged combat."
```
Don't mind me, just updating this to work with latest dev properly.